package.json

```diff
@@ -16,76 +7,57 @@
     "type": "git",
     "url": "https://github.com/skatejs/skatejs"
   },
-  "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
```

Comment:
perfect, actually SystemJS doesn't care as you can configure it whatever you want ( also it uses Rollup under the hood so I guess it's default behaviour is like rollups ).  

Thanks for clarifying!